src/Sleep.js

```diff
@@ -0,0 +1,106 @@
+class Sleep {
+  constructor(sleepData) {
+    this.sleepData = sleepData;
+  }
+
+  getUserData(id) {
+    return this.sleepData.filter(obj => {
+      return obj.userID === id;
+    });
+  }
+
+  calculateAverageHours(id) {
+    return this.getUserData(id).reduce((acc, object) => {
+      return Math.round(acc += object.hoursSlept / this.getUserData(id).length);
+    }, 0);
+  }
+
+  calculateAverageQuality(id) {
+    return this.getUserData(id).reduce((acc, object) => {
+      return Math.round(acc += object.sleepQuality / this.getUserData(id).length);
+    }, 0);
+  }
```

Comment:
same issue as the previous method

Reply:
Ill go through these. This changes down the road. As i got better with it

Reply:
Same as above. Suggest we leave for now and refactor when we have everything working.